api_fhir_r4/mapping/hfOrganizationMapping.py

```diff
@@ -0,0 +1,31 @@
+from api_fhir_r4.configurations import R4LocationConfig
+from api_fhir_r4.models.imisModelEnums import ImisLocationType
+
+
+class HealthFacilityOrganizationTypeMapping:
+    LEGAL_FORM_CODE = 'D'
+    LEGAL_FORM_DISPLAY = 'District organization'
+    LEGAL_FORM_SYSTEM = 'https://openimis.github.io/openimis_fhir_r4_ig/CodeSystem/organization-legal-form'
+    LEGAL_FORM_URL = 'https://openimis.github.io/openimis_fhir_r4_ig/StructureDefinition/organization-legal-form'
+
+    ORGANIZATION_TYPE = 'prov'
+
+    EMAIL_CONTACT_POINT_SYSTEM = 'email'
```

Comment:
These systems are also used by Patient profile. Should be reused.

Reply:
Currently Patient converter is using the hardcoded string value  ([Link](https://github.com/openimis/openimis-be-api_fhir_r4_py/blob/0b6601174b6a22b10e52f3042e030625e6d793a2/api_fhir_r4/converters/personConverterMixin.py#L38)). I seen that in previous version of FHIR API generic [ContactPointSystem enum](https://github.com/openimis/openimis-be-api_fhir_py/blob/e8541646354c0e3aa57340a3d8c5718606dbce4a/api_fhir/models/contactPoint.py#L15) was used. Should we go with same approach in here?

Reply:
I've added new enum classes for Contact Point.